src/api.js

```diff
@@ -0,0 +1,28 @@
+var url = 'https://en.wikipedia.org/w/api.php';
+
+var params = {
+  action: 'query',
+  generator: 'geosearch',
+  prop: 'coordinates|pageimages',
+  ggscoord: '37.7891838|-122.4033522',
+  format: 'json',
+};
+
+url = url + '?origin=*';
+Object.keys(params).forEach(function (key) {
+  url += '&' + key + '=' + params[key];
+});
+
+fetch(url)
+  .then(function (response) {
+    return response.json();
+  })
+  .then(function (response) {
+    var pages = response.query.pages;
+    for (var page in pages) {
+      console.log(response.json());
+    }
```

Comment:
Wait, is this file even being used? 😅 

I expected to see a `return` statement here, then I went looking for where you import `api.js` but didn't see it anywhere. Is this file in use?

Reply:
If the file isn't being imported anywhere, please delete it. As you can see, I spent quite a bit of time reviewing a file that I'm pretty sure isn't even in use. That's...not great.

Reply:
Apologies and noted! I was using this as a working file to get some logic down and forgot to remove